Userland/Utilities/pmemdump.cpp

```diff
@@ -0,0 +1,116 @@
+/*
+ * Copyright (c) 2021, Andrew Kaster <akaster@serenityos.org>
```

Comment:
I didn't write this 😬

Reply:
Oops, I copied the fdtdump file as a template just to start somewhere... thank you for noticing!